src/components/Header/__snapshots__/Header.spec.jsx.snap

```diff
@@ -1,41 +1,96 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`Header should display children 1`] = `
-<div
-  className="HeaderColor_default"
->
-  content
-</div>
+Array [
+  <div
+    className=""
+  >
+    <div
+      className="HeaderColor_normal"
+    >
+      content
+    </div>
+  </div>,
+  <div
+    className="HeaderColor_normal"
+  >
+    content
+  </div>,
+]
```

Comment:
I don't understand why we have an array with twice the content here?

Reply:
Good catch. A bad bit of inattention on my part. The reason is simple actually : as I was `find`ing the `divs` in the `setup`, two divs were returned (one child of the other), thus causing two elements in the array.